README.md

```diff
@@ -54,9 +56,42 @@ To increase logging level to include info and debug messages, set the environmen
 `DMCI_LOGLEVEL` to the desired level. Valid levels are `CRITICAL`, `ERROR`, `WARNING`, `INFO`, and
 `DEBUG`.
 
+## Installation
+```
+git clone https://github.com/metno/discovery-metadata-catalog-ingestor
+
+cd discovery-metadata-catalog-ingestor
+
+mkdir workdir
+
+cd workdir
+
+mkdir rejected
+
+```
+Initialize the config.yaml or use environment-variables.
```

Comment:
it took me a while to understand that "initialize the config.yaml" apparently means just create the file based on the example-config.yaml and fill in with the following .. maybe we can say that explicitly? (or maybe this is too dumb, idk)

Reply:
yes, please be explicit :)